compliance_checker/cf/cf_1_7.py

```diff
@@ -237,9 +237,12 @@ def check_cell_boundaries(self, ds):
         # Note that test does not check monotonicity
         ret_val = []
         reasoning = []
+
         for variable_name, boundary_variable_name in cfutil.get_cell_boundary_map(
             ds
         ).items():
+            
+            print(boundary_variable_name, boundary_variable_name.dtype)
```

Comment:
Please remove print statement.  If needed for debug purposes, please use an appropriate low logging level such as `DEBUG` to suppress from normal output in the compliance checker.

Reply:
removed "print( )", it was used for debugging.